pkgs/servers/zigbee2mqtt/default.nix

```diff
@@ -3,14 +3,15 @@ let
   package = (import ./node.nix { inherit pkgs system; }).package;
 in
 package.override rec {
-  version = "1.16.1";
+  version = "1.16.2";
   reconstructLock = true;
 
   postInstall = ''
     sed -i '1s;^;#!/usr/bin/env node\n;' $out/lib/node_modules/zigbee2mqtt/index.js
-    chmod +x $out/lib/node_modules/zigbee2mqtt/index.js
+    chmod +x $out/lib/node_modules/zigbee2mqtt/cli.js
+    rm -rf $out/bin
```

Comment:
What's in here that you need to delete?

Reply:
The bin created by npm: https://github.com/Koenkk/zigbee2mqtt/blob/1.16.2/package.json#L79